### ISSUE TITLE
EasyAutocomplete does not support mobile browsers (tested on Samsung S3+/Chrome version 33.0.1750.136 and LG G3S/stock browser)

### ISSUE THREAD

**Issue by meijastiina** [pawelczak/EasyAutocomplete#117]
Seems that EasyAutocomplete does not work on mobile browsers. I tested it with your demo (http://easyautocomplete.com). The autocomplete list does not appear when typing even though there are matches.

Repro steps:
1. Open browser on your mobile phone
2. Go to http://easyautocomplete.com 
3. Type 'a' on the autocomplete input
4. Observe results

Expected result: A list with matching items (Afghanistan, Albania etc) to appear automatically beneath the input box as in desktop browser.

Actual result: Nothing happens.

**Comment by nuno-bento** [comment]
Yeah, I have got the same problem on the iPhone. Any idea when this plugin will work on mobiles?

**Comment by quangthien27** [comment]
This also happen with mobile Firefox on Android

**Comment by Dr-Mx** [comment]
Referencing https://github.com/pawelczak/EasyAutocomplete/issues/189

**Comment by maba4891** [comment]
same  here! (tested with current chrome and firefox on android and safari on an iphone4)
Makes this plugin pretty useless as we see a lot of mobile traffic

**Comment by Dr-Mx** [comment]
Try https://github.com/pawelczak/EasyAutocomplete/issues/189#issuecomment-274885047
